### PR TITLE
[Backport 2024.01.xx] Bump axios from 0.24.0 to 0.28.0 #9985

### PR DESCRIPTION
**Pull request by offtherailz**
[Backport 2024.01.xx] Bump axios from 0.24.0 to 0.28.0 #9985

### CODE REVIEW

File: .github/workflows/update_gs_versions.yml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ github.event.inputs.main-branch }}
+          token: ${{ secrets.GITHUB_TOKEN }}
+      - name: Update versions
+        id: create-versions
+        env:
+          MAPFISH_GROUP: org.mapfish.print
```
Comment (allyoucanmap) [inline .github/workflows/update_gs_versions.yml:27]: Probably this file should not be included

File: web/client/components/map/cesium/__tests__/Map-test.jsx [hunk 2]
```diff
@@ -382,7 +382,27 @@ describe('CesiumMap', () => {
...
+            if (currentReady !== prevReady) {
+                if (currentReady) {
+                    countReady += 1;
+                }
+                prevReady = ref.map.dataSourceDisplay.ready;
+            }
+            if (countReady === 2) {
+                return true;
+            }
+            return false;
```
Comment (allyoucanmap) [inline web/client/components/map/cesium/__tests__/Map-test.jsx:400]: Probably this file should not be included

**Comment by offtherailz** [comment]
I pointed the wrong target for the PR. Now is for 2024.01.xx

**Review by allyoucanmap — APPROVED** [review]

**Comment by allyoucanmap** [comment]
@ElenaGallo please test this fix on QA, thanks

**Pull request merged**